File: Facturador/app/src/main/java/com/example/facturador/Activity/Model/DetalleEquipo.java
package com.example.facturador.Activity.Model;

import java.io.Serializable;

public class DetalleEquipo  implements Serializable {

    private int iddetalle_equipo;
    private int idequipo;
    private int idcomercio;
    private int id_base;

    public DetalleEquipo() {

    }

    public int getId_base() {
        return id_base;
    }

    public void setId_base(int id_base) {
        this.id_base = id_base;
    }

    public DetalleEquipo(int iddetalle_equipo, int idequipo, int idcomercio) {
        this.iddetalle_equipo = iddetalle_equipo;
        this.idequipo = idequipo;
        this.idcomercio = idcomercio;
    }

    public int getIddetalle_equipo() {
        return iddetalle_equipo;
    }

    public void setIddetalle_equipo(int iddetalle_equipo) {
        this.iddetalle_equipo = iddetalle_equipo;
    }

    public int getIdequipo() {
        return idequipo;
    }

    public void setIdequipo(int idequipo) {
        this.idequipo = idequipo;
    }

    public int getIdcomercio() {
        return idcomercio;
    }

    public void setIdcomercio(int idcomercio) {
        this.idcomercio = idcomercio;
    }
}

File: webservices/CORE/DetalleEquipo.php
<?php


class DetalleEquipo
{
    public $iddetalle_equipo,$idequipo,$idcomercio,$estado;

    /**
     * @return mixed
     */
    public function getEstado()
    {
        return $this->estado;
    }

    /**
     * @param mixed $estado
     */
    public function setEstado($estado)
    {
        $this->estado = $estado;
    }

    /**
     * DetalleEquipo constructor.
     * @param $iddetalle_equipo
     * @param $idequipo
     * @param $idcomercio
     */
    public function __construct($iddetalle_equipo, $idequipo, $idcomercio)
    {
        $this->iddetalle_equipo = $iddetalle_equipo;
        $this->idequipo = $idequipo;
        $this->idcomercio = $idcomercio;
    }

    /**
     * @return mixed
     */
    public function getIddetalleEquipo()
    {
        return $this->iddetalle_equipo;
    }

    /**
     * @param mixed $iddetalle_equipo
     */
    public function setIddetalleEquipo($iddetalle_equipo)
    {
        $this->iddetalle_equipo = $iddetalle_equipo;
    }

    /**
     * @return mixed
     */
    public function getIdequipo()
    {
        return $this->idequipo;
    }

    /**
     * @param mixed $idequipo
     */
    public function setIdequipo($idequipo)
    {
        $this->idequipo = $idequipo;
    }

    /**
     * @return mixed
     */
    public function getIdcomercio()
    {
        return $this->idcomercio;
    }

    /**
     * @param mixed $idcomercio
     */
    public function setIdcomercio($idcomercio)
    {
        $this->idcomercio = $idcomercio;
    }


}